ci/jenkins/Test.groovy.j2

```diff
@@ -199,6 +217,9 @@ stage('Test') {
     {{ method_name }}()
   },
   {% endfor %}
+  'unittest: CPU MINIMAL': {
+    run_unittest_minimal()
+  },
```

Comment:
I still don't understand why we need a new macro, does this change not do the same thing with existing code?
```suggestion
  {% call m.test_step(
    name="unittest: CPU MINIMAL",
    node="CPU-SMALL",
    ws="tvm/ut-python-cpu-minimal",
    platform="minimal",
    docker_image="ci_minimal",
  ) %}
    {{ m.download_artifacts(tag='cpu-minimal', filenames=tvm_lib) }}
    cpp_unittest(ci_minimal)
    python_unittest(ci_minimal)
  {% endcall %}
```

Reply:
Not really. The `test_step` macro will expand and generate something as if the body of run_unittest_minimal was inlined at the call site. 

Due to the JVM method size limit, I cannot inline the body of `run_unittest_minimal` inside the Jenkinsfile. 

Therefore, I need a macro that does the same thing as `test_step` but does not include the declaration that is part of the `parallel`:  `unittest: CPU MINIMAL: {`